css/main.css

```diff
@@ -0,0 +1,294 @@
+/* font-family: 'Inter', sans-serif; */
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+:root {
+  --clr-primary: #36b37f;
+  --clr-dark: #172b4d;
+  --clr-light: #ebf0ee;
+  --fw-normal: 400;
+  --fw-lightBold: 500;
+  --fw-blod: 600;
+  --fw-black: 700;
+  --grid: 1fr;
+}
+
+body {
+  margin: 0;
+  font-family: 'Inter', sans-serif;
+  font-size: 1.75rem;
+  color: var(--clr-dark);
+  line-height: 1.4;
+  background-color: white;
+  background-image: url(../images/background/header-illsutration-mobile.svg);
+  background-repeat: repeat-y;
+}
+
```

Comment:
- [ ] Good job thus far, however, there is something you need to fix in order for your project to look fantastic, but you are almost there, as we copy the Figma design and if we compare your design to the Figma design, there are differences. Please update your design for a good vision :wink: 
Your Design :point_down: 
![masta](https://user-images.githubusercontent.com/88894525/192120613-dba767c5-fffc-4d9c-9abb-7f51b9a1e575.png)

Figma Design :point_down: 
![mastafigma](https://user-images.githubusercontent.com/88894525/192120630-6eabe69e-b707-4ed0-a6e0-687fbca37481.png)